tests/test_plotting.py

```diff
@@ -128,5 +184,15 @@ def test_legend(self):
         self.assertEqual(labels, expected)
 
 
+def _check_colors(collection, expected_colors):
+
+    import matplotlib.colors as colors
+    conv = colors.colorConverter
+
+    for patch, color in zip(collection, expected_colors):
+        result = patch.get_color()
+        assert conv.to_rgba(result) == conv.to_rgba(color)
```

Comment:
Thanks for looking into this.

The numpy testing assertions work fine with tuples (they take "array_like" args), and will handle the floating point comparisons. It's probably the cleanest way to do this, though I'm open to alternatives. (We agree that this not relevant to the current test failure anyway.)